core/webapp/vis/src/statistics.js

```diff
@@ -199,20 +199,50 @@ LABKEY.vis.Stat.MEAN = LABKEY.vis.Stat.getMean;
 /**
  * Returns the standard deviation.
  * @param values An array of numbers.
+ * @param sample If true, use (n-1) for the denominator of the final step. Defaults to false.
  * @returns {Number}
  */
-LABKEY.vis.Stat.getStdDev = function(values)
+LABKEY.vis.Stat.getStdDev = function(values, sample)
 {
+    if (sample === undefined || sample === null) sample = false;
```

Comment:
This conversion to false shouldn't be necessary for two reasons.

1) null and undefined are falsey, and the checks below are compatible with falsey values.
2) we can use default arguments, they are supported all major browsers that we support.
```suggestion
LABKEY.vis.Stat.getStdDev = function(values, sample = false) {
```

Reply:
done